sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_component_func.py

```diff
@@ -38,6 +50,15 @@ def get_dynamic_source_parameter(source):
 
 
 def to_component_func(entity: ComponentEntity, component_creation_func) -> Callable[..., Command]:
+    """Convert a ComponentEntity to a callable component function.
+
+    :param entity: The ComponentEntity to convert.
+    :type entity: ComponentEntity
```

Comment:
Can you change this to ~azure.ai.ml.entities.Component?

ComponentEntity isn't a real class. It's Component being imported and aliased as "ComponentEntity".

`from azure.ai.ml.entities import Component as ComponentEntity`

Reply:
done